packages/shopify-codemod/test/fixtures/split-return-assignments/basic.output.js

```diff
@@ -0,0 +1,5 @@
+foo(() => {
+  console.log(bar);
+  this.sho = this.doo;
+  return this.sho;
+});
```

Comment:
Yeah, any group of assignments should be handled (they will all be handled the same way anyways).